packages/core/helper-plugin/src/hooks/useClipboard.js

```diff
@@ -0,0 +1,33 @@
+import { useCallback } from 'react';
+
+export const useClipboard = () => {
+  const copy = useCallback(async (value) => {
+    try {
+      // only strings and numbers casted to strings can be copied to clipboard
+      if (typeof value !== 'string' && typeof value !== 'number') {
+        throw new Error(`Cannot copy typeof ${typeof value} to clipboard, must be a string`);
+      }
+      // empty strings are also considered invalid
+      else if (value === '') {
+        throw new Error(`Cannot copy empty string to clipboard.`);
+      }
+
+      const stringifiedValue = value.toString();
+
+      await navigator.clipboard.writeText(stringifiedValue);
+
+      return true;
+    } catch (error) {
+      /**
+       * Realistically this isn't useful in production as there's nothing the user can do.
+       */
+      if (process.env.NODE_ENV === 'development') {
+        console.warn('Copy failed', error);
+      }
```

Comment:
Should we display a notification instead saying the browser either does not support it or permissions are missing?

Reply:
Hmmm I'd prefer not to cross pollute this hooks with a dependency of notification... we do return false if it fails so in theory in the components can handle independently what they want to do. We could make the API more advance to handle all these edge cases if you see benefit? 😄

Reply:
Makes sense!